src/main/java/no/nav/pto/veilarbportefolje/persononinfo/PdlPersonRepository.java

```diff
@@ -36,21 +67,83 @@ public void slettLagretBrukerData(List<Fnr> identer) {
         }
         String identerParam = identer.stream().map(Fnr::get).collect(Collectors.joining(",", "{", "}"));
         db.update("DELETE from bruker_data where freg_ident = any (?::varchar[])", identerParam);
+        db.update("DELETE FROM BRUKER_STATSBORGERSKAP WHERE FREG_IDENT = any (?::varchar[]) ", identerParam);
     }
 
     @SneakyThrows
     public PDLPerson hentPerson(Fnr hentAktivFnr) {
+        List<Statsborgerskap> statsborgerskaps = hentStatsborgerskap(hentAktivFnr);
         return queryForObjectOrNull(() ->
-                db.queryForObject("select * from bruker_data where freg_ident = ?",
+                dbReadOnly.queryForObject("select * from bruker_data where freg_ident = ?",
                         (rs, row) -> new PDLPerson()
                                 .setFornavn(rs.getString("fornavn"))
                                 .setEtternavn(rs.getString("etternavn"))
                                 .setMellomnavn(rs.getString("mellomnavn"))
                                 .setKjonn(Kjonn.valueOf(rs.getString("kjoenn")))
                                 .setErDoed(rs.getBoolean("er_doed"))
                                 .setFoedsel(rs.getDate("foedselsdato").toLocalDate())
-                        ,
-                        hentAktivFnr.get())
-        );
+                                .setFoedeland(rs.getString("foedeland"))
+                                .setInnflyttingTilNorgeFraLand(rs.getString("innflyttingTilNorgeFraLand"))
+                                .setAngittFlyttedato(toLocalDateOrNull(rs.getString("angittFlyttedato")))
+                                .setTalespraaktolk(rs.getString("talespraaktolk"))
+                                .setTegnspraaktolk(rs.getString("tegnspraaktolk"))
+                                .setTolkBehovSistOppdatert(toLocalDateOrNull(rs.getString("tolkBehovSistOppdatert")))
+                                .setStatsborgerskap(statsborgerskaps)
+                        , hentAktivFnr.get()));
+    }
+
+    private List<Statsborgerskap> hentStatsborgerskap(Fnr fnr) {
+        return dbReadOnly.query("""
+                    SELECT STATSBORGERSKAP, GYLDIG_FRA, GYLDIG_TIL FROM BRUKER_STATSBORGERSKAP WHERE FREG_IDENT = ?
+                """, (rs, rowNum) -> new Statsborgerskap(
+                rs.getString("STATSBORGERSKAP"),
+                toLocalDateOrNull(rs.getString("GYLDIG_FRA")),
+                toLocalDateOrNull(rs.getString("GYLDIG_TIL"))
+        ), fnr.get());
+    }
+
+    public Map<Fnr, List<Statsborgerskap>> hentStatsborgerskap(List<Fnr> fnrs) {
```

Comment:
Maybe change the name to: hent**Gyldige**StatsborgerskapData
as this method does a filter which is not done in the method above